ci/azure-macos-35.yaml

```diff
@@ -11,7 +11,7 @@ dependencies:
   - matplotlib
   - nomkl
   - numexpr
-  - numpy=1.10.4
+  - numpy=1.12.1
```

Comment:
Most of the code / docs point to 1.12.0 but this is 1.12.1 - any reason for the minor version bump here?

Reply:
Nope, that was an oversight...